src/Application.js

```diff
@@ -26,6 +26,8 @@ const completionPercentage = 'completionPercentage';
 const speedScale = 'speedScale';
 const timersScale = 'timersScale';
 const inputCount = 'inputCount';
+const fullScreen = 'fullScreen';
```

Comment:
Just looks like some of the fullScreen stuff made it in here, seems to just be the feature names and stuff. Should be removed to keep the two PRs separate